### DISCUSSION TITLE
Change table tenants name

### DISCUSSION THREAD

**Discussion by awebapps** [spatie/laravel-multitenancy#417] [Q&A] [answered]
Hi dear friends..

I'd like to change a name from tenants table in the database landlord When I change it to tenantgroups I get the following error SQLSTATE[42S02]: Base table or view not found: 1146 Table 'landlord.tenants' doesn't exist (SQL: select * from `tenants` where `domain` = shop1.local limit 1) I can't find how to change it.

Thanks for help.....

**Comment by masterix21** [comment, marked as answer]
It would be best if you extended the package `Tenant` model specifying the database table like so:
```php
class TenantGroup extends \Spatie\Multitenancy\Models\Tenant
{
    protected $table = 'tenantgroups';
}
```

**Reply by awebapps** [reply]
Hi
Thank you for helping me..
I don't know where to put this script..

Thanks for the help again
